code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -69,31 +69,141 @@
 	permeability_coefficient = 0.9
 	siemens_coefficient = 0.9
 
+//Special type of gloves. Alt click and you get some special nodrop gloves
+/obj/item/clothing/gloves/heldgloves
+	name = "gloves"
+	var/obj/item/weapon/heldglove/right_glove = /obj/item/weapon/heldglove
+	var/obj/item/weapon/heldglove/left_glove = /obj/item/weapon/heldglove
+
+/obj/item/clothing/gloves/heldgloves/examine(mob/user)
+	. = ..()
+	to_chat(user, "<span class='notice'>Alt-click the gloves when worn to strap them into your hands.")
+
+/obj/item/clothing/gloves/heldgloves/unequipped(mob/unequipper, slot)
+	. = ..()
+	remove_gloves(unequipper)
+
+//We use alt-click to activate/deactive the gloves in-hand
+/obj/item/clothing/gloves/heldgloves/AltClick(mob/user)
+	var/mob/living/carbon/human/wearer = user
+	if(wearer.gloves != src) //We have to be wearing the gloves first
+		return
+
+	if(!remove_gloves(user))
+		return
+
+	user.drop_all_held_items() //Gloves require free hands
+	create_gloves(user)
+
+/// Creates the held items for user and puts it in their hand
+/obj/item/clothing/gloves/heldgloves/proc/create_gloves(mob/user)
+	if(user.l_hand != null || user.r_hand != null)
+		return
+
+	var/obj/item/weapon/heldglove/boxing/rightglove = new right_glove
+	user.put_in_r_hand(rightglove, TRUE)
 
+	var/obj/item/weapon/heldglove/boxing/leftglove = new left_glove
+	user.put_in_l_hand(leftglove, TRUE)
+	return
 
-/obj/item/clothing/gloves/boxing
+/// Removes gloves. Returns false if gloves are not currently worn
+/obj/item/clothing/gloves/heldgloves/proc/remove_gloves(mob/user)
+	if(istype(user.l_hand, /obj/item/weapon/heldglove))
+		qdel(user.l_hand)
+		if(istype(user.r_hand, /obj/item/weapon/heldglove))
+			qdel(user.r_hand)
+			return FALSE
+	if(istype(user.r_hand,/obj/item/weapon/heldglove))
+		qdel(user.r_hand)
+		return FALSE
+	return TRUE
+
+/obj/item/weapon/heldglove
+	name = "glove"
+
+
+//Boxing gloves
+/obj/item/clothing/gloves/heldgloves/boxing
 	name = "boxing gloves"
 	desc = "Because you really needed another excuse to punch your crewmates."
 	icon_state = "boxing"
 	item_state = "boxing"
+	left_glove = /obj/item/weapon/heldglove/boxing/hook
```

Comment:
no, either it is a object or a path. You are using it as an object, so don't do that
```suggestion
```